### PR TITLE
Add default capability set for new version

### PR DESCRIPTION
**Pull request by yunjiang29**
/hold

### CODE REVIEW

File: ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh [hunk 1]
```diff
@@ -37,6 +37,7 @@ echo "OCP Version: $ocp_version"
 
 v411="baremetal marketplace openshift-samples"
 v412="baremetal marketplace openshift-samples Console Insights Storage CSISnapshot"
+v413="baremetal marketplace openshift-samples Console Insights Storage CSISnapshot"
```
Comment (jianlinliu) [inline ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh:40]: How about we always set the default of the new version which equal to the last version's unless we get know the newly added operators.

File: ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh [hunk 2]
```diff
@@ -63,6 +67,9 @@ case ${BASELINE_CAPABILITY_SET} in
 "v4.12")
   base_operators="${v412}"
   ;;
+"v4.13")
+  base_operators="${v413}"
```
Comment (jianlinliu) [inline ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh:71]: How about renaming the var name to `enabled_operators` for easy reading.

File: ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh [hunk 3]
```diff
@@ -77,11 +84,21 @@ echo "Baseline Capability Set: $base_operators"
...
-    fi
-fi
+case ${ocp_version} in
+"4.12")
+  if [[ ! "${all_caps}" =~ "Storage" ]]; then
+      namespace="openshift-cluster-csi-drivers"
+      remove_secrets "${SHARED_DIR}" "${namespace}" || exit 1
+  fi
+  ;;
+"4.13")
```
Comment (jianlinliu) [inline ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh:94]: Is there other solution, that can help us not have to change the code when a new version come out.

File: ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh (hunk 1 above)
Comment (yunjiang29) [reply ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh]: updated.

File: ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh (hunk 2 above)
Comment (yunjiang29) [reply ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh]: updated.

File: ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh (hunk 3 above)
Comment (yunjiang29) [reply ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh]: updated.

File: ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh [hunk 4]
```diff
@@ -28,60 +27,70 @@ function remove_secrets()
...
+enabled_operators=""
 case ${BASELINE_CAPABILITY_SET} in
 "None")
   ;;
-"v4.11")
-  base_operators="${v411}"
-  ;;
-"v4.12")
-  base_operators="${v412}"
+"${v_current_version_with_dot}")
```
Comment (jianlinliu) [inline ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh:70]: We also support to install 4.N cluster with `BASELINE_CAPABILITY_SET=4.M` (M<N)

Comment (jianlinliu) [reply ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh]: @yunjiang29 how about like this:
```
if [[ "${BASELINE_CAPABILITY_SET}" == "" ]]; then
  echo "This step is not required when BASELINE_CAPABILITY_SET is not set"
  exit 0
fi

if (( ocp_minor_version <=10 && ocp_major_version == 4 )) || (( ocp_major_version < 4 )); then
  echo "This step is not required for ${ocp_version}, exit now"
  exit 0
fi

v411="baremetal marketplace openshift-samples"
# shellcheck disable=SC2034
v412="${v411} Console Insights Storage CSISnapshot"
latest_defined="v412"
always_default="${!latest_defined}"

declare "v${ocp_major_version}${ocp_minor_version}"
v_current_version="v${ocp_major_version}${ocp_minor_version}"

if [[ ${!v_current_version} == "" ]]; then
  echo "No default value for ${v_current_version}, use default value from ${latest_defined}: ${always_default}"
  vCurrent=${always_default}
else
  vCurrent=${!v_current_version}
fi
echo "vCurrent set: $vCurrent"

# Base Capability
enabled_operators=""
case ${BASELINE_CAPABILITY_SET} in
"None")
  ;;
"v4.11")
  enabled_operators="${v411}"
  ;;
"v4.12")
  enabled_operators="${v412}"
  ;;
"vCurrent")
  enabled_operators="${vCurrent}"
  ;;
*)
  enabled_operators="${always_default}"
  ;;
esac
```

**Pull request merged**